finality-aleph/src/abft/mod.rs

```diff
@@ -72,6 +73,24 @@ impl<S: 'static> IntoIterator for SignatureSet<S> {
     }
 }
 
+impl From<SignatureSet<crate::crypto::Signature>>
```

Comment:
We generally prefer top level imports (perhaps renaming) in the codebase to inline module mentions. This could just be `Signature` and below also `PrimitivesSignatureSet` or something.

Reply:
changed